# Paired sample z-test

[Paired-sample z-test](http://facweb.cs.depaul.edu/sjost/csc423/documents/test-descriptions/paired-z.pdf "Paired-sample z-test")

Assumptions

• Experimental Design: The sample forms two treatment groups,
where each subject in one group is paired with a subject in the other
group.

• Null Hypothesis: The average of the pair differences is not significantly different than zero.

• Population Distribution: Arbitrary.

• Sample Size: The sample size of each treatment group is greater than
or equal to 30


## Case 1 one tailed z test

In a kindergarten, 49 children were tested with Binet intelligence test (σ = 16) when they entered the kindergarten. The average intelligence quotient was X1 = 106. One year later, the same group of subjects were tested again. The result was x2 = 110. The correlation coefficient of the two test results was known to be r = 0.74. Can you say that the children's intelligence quotient increased significantly with the age and one year's education. (alpha =0.01)


H0 : ${\mu_1}$ $\geq$ ${\mu_2}$
<br/>
H1 : ${\mu_1}$ 	$<$ ${\mu_2}$

$\bar{X_1}$ = 106, $\bar{X_2}$ = 110, $\sigma_1$ = 16, $\sigma_2$ = 16, $n$ = 49, $\rho$ = 0.74

z = $\frac{\bar{X_1}-\bar{X_2}}{ \sqrt{\frac{ \sigma_1^2 + \sigma_2^2 - 2*\rho*\sigma_1*\sigma_2}{n}}}$ = $\frac{106-110}{ \sqrt{\frac{ 16^2 + 16^2 - 2*0.74*16*16}{49}}}$


In [1]:
import math
z = (106 - 110) / math.sqrt((math.pow(16, 2) + math.pow(16, 2) - 2*0.74*16*16)/49)
print(z)

-2.426813358485377



α = 0.01, look up table $Z_{\alpha}$ = 2.33

$\lfloor z \lfloor$ = 2.43 > 2.33, so reject H0 and accept H1, we think that children's IQ has improved significantly after one year